proto/palomachain/paloma/scheduler/tx.proto

```diff
@@ -3,16 +3,32 @@ package palomachain.paloma.scheduler;
 
 import "palomachain/paloma/scheduler/job.proto";
 import "palomachain/paloma/valset/common.proto";
+import "palomachain/paloma/scheduler/params.proto";
 import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
 
 option go_package = "github.com/palomachain/paloma/x/scheduler/types";
 
 // Msg defines the Msg service.
 service Msg {
   rpc CreateJob(MsgCreateJob) returns (MsgCreateJobResponse);
   rpc ExecuteJob(MsgExecuteJob) returns (MsgExecuteJobResponse);
+  rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse);
 }
 
+message MsgUpdateParams {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+  // params defines the x/scheduler parameters to update.
+  Params params = 2 [(gogoproto.nullable) = false];
+}
+
+message MsgUpdateParamsResponse {}
+
```

Comment:
Do we really need this now? Paloma isn't really using params yet.

If so, please make sure you add the `Metadata` field that all other Paloma messages are embedding.

Reply:
removed and pushed